### ISSUE TITLE
[Bug] error when call subcommand help

### ISSUE THREAD

**Issue by XDean** [c4spar/deno-cliffy#154]
Following code is from document https://deno.land/x/cliffy@v0.17.2/command#%E2%9D%AF-commands

```
import { Command } from "https://deno.land/x/cliffy/command/mod.ts";

const clone = new Command()
  .arguments("<source:string> [destination:string]")
  .description("Clone a repository into a newly created directory.")
  .action((options: any, source: string, destination: string) => {
    console.log("clone command called");
  });

await new Command()
  .command("clone", clone)
  .parse(Deno.args);
```

then show help: ` deno run --unstable mod.ts clone -h`

error shown:

```
error: Uncaught TypeError: this._help is not a function
    return (this._help!)();
                        ^
    at Command.getHelp (https://deno.land/x/cliffy@v0.17.2/command/command.ts:1166:25)
    at Command.showHelp (https://deno.land/x/cliffy@v0.17.2/command/command.ts:1160:57)
    at Command.error (https://deno.land/x/cliffy@v0.17.2/command/command.ts:1051:10)
    at Command.parse (https://deno.land/x/cliffy@v0.17.2/command/command.ts:735:18)
    at Command.parse (https://deno.land/x/cliffy@v0.17.2/command/command.ts:682:33)
    at file:///D:/coding/codes/deno-create-react-app/mod.ts:12:4
```

**Comment by c4spar** [comment]
Hi @XDean, thx for the issue!

The issue should be already fixed in a separate branch.
```
import { Command } from "https://raw.githubusercontent.com/c4spar/deno-cliffy/command/generic-options/command/mod.ts";

const clone = new Command()
  .arguments("<source:string> [destination:string]")
  .description("Clone a repository into a newly created directory.")
  .action((options: any, source: string, destination: string) => {
    console.log("clone command called");
  });

await new Command()
  .command("clone", clone)
  .parse(Deno.args);
```
I will release a new version later or tomorrow.

**Comment by c4spar** [comment]
Hey @XDean,
sry, it took a bit longer, but the fix is available in v0.18.0 now.
